docker/Dockerfile

```diff
@@ -21,6 +21,7 @@ ARG INDEX_URL=""
 ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
 ENV PATH="$PATH:/usr/local/airflow/.local/bin:/root/.local/bin:/usr/local/airflow/.local/lib/python3.10/site-packages"
 ENV PYTHON_VERSION=3.11.6
+ENV AIRFLOW_VERSION=2.7.2
```

Comment:
nitpick: `ENV AIRFLOW_VERSION=$AIRFLOW_VERSION` to reduce any source for error and avoid having the same magic-number defined in two places.

Reply:
Using curly brackets to keep consistency with `AIRFLOW_HOME`